packages/loopring_v3/contracts/core/impl/libexchange/ExchangeDeposits.sol

```diff
@@ -46,26 +47,53 @@ library ExchangeDeposits
 
         uint16 tokenID = S.getTokenID(tokenAddress);
 
-        // Transfer the tokens to this contract
-        uint96 amountDeposited = S.depositContract.deposit{value: msg.value}(
-            from,
-            tokenAddress,
-            amount,
-            extraData
-        );
+        uint96 amountDeposited = amount;
+        if (flash) {
+            require(msg.value == 0, "INVALID_FLASH_DEPOSIT");
+            S.amountFlashDeposited[tokenAddress] = S.amountFlashDeposited[tokenAddress].add(amount);
```

Comment:
maybe we should credit these deposits to an operator account *explicitly* instead of allowing the relayer to credit them to any L2 account.

Reply:
Is there a benefit in doing that? It doesn't seem to matter to me who which account receives the tokens, and the implementation would remain the same as far as I can see.